### DISCUSSION TITLE
export and/or backup the data

### DISCUSSION THREAD

**Discussion by rafaelrdealmeida** [omnivore-app/omnivore#3529] [Q&A] [answered]
Hello, is it possible to export and/or backup the data?

**Comment by jacksonh** [comment, marked as answer]
Hi, what most people do is use something like Obsidian to syncronize their data to local files as markdown. We have a plugin that Obsidian and Logseq can use to pull down data and format it how you want for long term storage.

**Reply by rafaelrdealmeida** [reply]
thanks!

**Comment by jacksonh** [comment]
I'll close this as there is _something_ in the future we definitely need more backup tools

**Reply by chansantheman** [reply]
Yes, I am have troubling find sensible export options such as csv how Upnext gave when it sunsetted.

**Reply by candideu** [reply]
Here's one I used and liked: https://github.com/Cito/omnivore-export
